Controller/OpenTemplateController.php

```diff
@@ -4,7 +4,7 @@
 
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpKernel\Debug\FileLinkFormatter;
+use Symfony\Component\ErrorHandler\ErrorRenderer\FileLinkFormatter;
```

Comment:
This change makes the controller incompatible with previous versions of symfony. As composer.json states, we have to support both variants of the file formatter.